core/arch/arm/include/mm/tee_mmu_defs.h

```diff
@@ -32,7 +32,7 @@
 #define TEE_MMU_TTBCR_N_VALUE		7
 
 /* Number of sections in ttbr0 when user mapping activated */
-#define TEE_MMU_UL1_NUM_ENTRIES         ((1 << TEE_MMU_TTBCR_N_VALUE) / 4)
+#define TEE_MMU_UL1_NUM_ENTRIES         (1 << (12 - TEE_MMU_TTBCR_N_VALUE))
```

Comment:
Ok,  i will test it , some DRM TAs need to deal with the video frame,  i need to map the video buffer into the TA VM space, 32M is not enough for 4K resolution videos.